stdlib/contrib/bonitoo-io/array/array.flux

```diff
@@ -0,0 +1,106 @@
+// Package array provides functions for interacting with arrays.
+package array
+
+
+import ejson "experimental/json"
+import "json"
+
+// append is a function that appends value to array.
```

Comment:
This description is misleading as it really append an array of values to another array of values.

Reply:
Should we also explicitly state that new memory space is allocated for this purpose, meaning this is not an efficient function?

Reply:
Fixed, thank you.